rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinTestUtils.java

```diff
@@ -29,6 +30,16 @@ public static List<BtcECKey> getBtcEcKeysFromSeeds(String[] seeds, boolean sorte
         return keys;
     }
 
+    public static List<ECKey> getEcKeysFromSeeds(String[] seeds) {
+        List<ECKey> keys = Arrays
+            .stream(seeds)
+            .map(seed -> ECKey.fromPrivate(
+                HashUtil.keccak256(seed.getBytes(StandardCharsets.UTF_8))))
+            .collect(Collectors.toList());
+
+        return keys;
+    }
+
```

Comment:
this probably shouldn't be here since the eckeys are not related to bitcoin But I didn't find a better place

Reply:
Looks like a good opportunity to finally create a utils class for Rsk related stuff.

`RskUtils`?

Reply:
ok!

Reply:
done. I created a `RskTestUtils` with just this method for now